### DISCUSSION TITLE
Backup automatico: si può?

### DISCUSSION THREAD

**Discussion by Aeiuoieoie** [globaleaks/globaleaks-whistleblowing-software#3959] [Q&A]
Buongiorno, premesso che dubito arriveranno a noi chissà quante segnalazioni, ho visto la procedura di backup per creare il tar.gz manualmente e dopo ogni update il backup è presente per 15 giorni. Ma se volessi automatizzare il backup una volta al mese per conservarlo, si può fare direttamente dalla parte amministrativa o si deve creare uno script automatico che richiami la procedura contenuta qui https://docs.globaleaks.org/en/main/user/admin/BackupAndRestore.html ?

**Comment by evilaliv3** [comment]
Al momento non è stato implementato alcun backup automatico.
Alcune ipotesi di sviluppo sono descritte sul ticketing system ma non credo ancora qualcuno ci stia lavorando attivamente.
